cockatrice/src/carddatabase.cpp

```diff
@@ -562,11 +551,10 @@ LoadStatus CardDatabase::loadFromFile(const QString &fileName, bool tokens)
                 if (xml.name() == "sets")
                     loadSetsFromXml(xml);
                 else if (xml.name() == "cards")
-                    loadCardsFromXml(xml, tokens);
+                    loadCardsFromXml(xml);
             }
         }
     }
-    qDebug() << cards.size() << "cards in" << sets.size() << "sets loaded";
```

Comment:
Yes, it's been merged with another similar debug line at https://github.com/Cockatrice/Cockatrice/pull/1859/files#diff-753bd6437004806e213fe65d32cfa732R606